app/scripts/controllers/permissions/snaps/keyManagementSnaps.ts

```diff
@@ -0,0 +1,62 @@
+export const KEY_MANAGEMENT_SNAPS = {
+  'a51ea3a8-f1b0-4613-9440-b80e2236713b': {
+    id: 'a51ea3a8-f1b0-4613-9440-b80e2236713b',
+    snapId: 'npm:@metamask/snap-simple-keyring',
+    iconUrl: '',
+    snapTitle: 'Metamask Simple Keyring',
+    snapSlug: 'Secure your account with MetaMask Mobile',
+    snapDescription:
+      'A simple private key is a randomly generated string of characters that is used to sign transactions. This private key is stored securely within this snap.',
+    tags: ['EOA'],
+    developer: 'Metamask',
```

Comment:
MetaMask is capitalized wrong

Reply:
file has been removed. we will update the registry repo to fix this.